ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/queue_monitor_length.schema.yml

```
@@ -13,6 +13,24 @@ keys:
         description: |
           "enabled: true" will be required in AVD4.0.
           In AVD3.x default is true as long as queue_monitor_length is defined and not None
+      default_thresholds:
+        type: dict
+        description: |
+          Set the default high and low thresholds for Etherent Interfaces
+        keys:
+          high:
+            type: int
+            required: true
+            convert_types:
+              - str
+            description: |
+              Set the default high threshold for Ethernet Interfaces
```

Suggesting to simplify this a bit.
```suggestion
              Default high threshold for Ethernet Interfaces.
```